apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.command.ts

```diff
@@ -1,3 +1,8 @@
+import { IsDefined, IsMongoId } from 'class-validator';
 import { OrganizationCommand } from '../../../shared/commands/organization.command';
 
-export class GetMyEnvironmentsCommand extends OrganizationCommand {}
+export class GetMyEnvironmentsCommand extends OrganizationCommand {
```

Comment:
I think you can use the existing `EnvironmentWithUserCommand` here instead of declaring `environmentId` manually here.

Reply:
I agree, check my last change